spring-cloud-dataflow-test/pom.xml

```diff
@@ -31,6 +31,16 @@
 			<groupId>org.testcontainers</groupId>
 			<artifactId>junit-jupiter</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-jdbc</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-csv</artifactId>
+			<version>1.10.0</version>
```

Comment:
If we decide to keep the test, we need to set the scope to test.

Reply:
Otherwise we increase the risk of potential hitch hiker CVEs.